spec/unit/config_files_renderer_spec.rb

```diff
@@ -119,6 +141,13 @@
           expect(output).to include("/etc/default/grub (grub2-2.00, mode)")
           expect(output).to include("/etc/postfix/main.cf (postfix-2.9.6, md5)")
         end
+
+         it "shows all known rpm changes" do
+          output = subject.render(system_description)
+
+          expect(output).to include("/etc/my.cnf (mariadb-5.5.33, size, mode, md5, device_number, " \
+            "link_path, user, group, time, capabilities)")
+        end
```

Comment:
`end` at 150, 8 is not aligned with `it "shows all known rpm changes" do` at 145, 9